Northstar.CustomServers/mod/scripts/vscripts/mp/_gamestate_mp.nut

```diff
@@ -177,17 +177,23 @@ void function GameStateEnter_PickLoadout()
 }
 
 void function GameStateEnter_PickLoadout_Threaded()
-{	
-	float pickloadoutLength = 20.0 // may need tweaking
-	SetServerVar( "minPickLoadOutTime", Time() + pickloadoutLength )
-	
-	// titan selection menu can change minPickLoadOutTime so we need to wait manually until we hit the time
-	while ( Time() < GetServerVar( "minPickLoadOutTime" ) )
-		WaitFrame()
-	
-	SetGameState( eGameState.Prematch )
-}
+{
+    float pickloadoutLength = 20.0 // may need tweaking
+    SetServerVar( "minPickLoadOutTime", Time() + pickloadoutLength )
+
+    // titan selection menu can change minPickLoadOutTime so we need to wait manually until we hit the time
+    while ( Time() < GetServerVar( "minPickLoadOutTime" ) )
+        WaitFrame()
 
+    //added by Khalmee for the sake of fixing LTS spectator mode
+    foreach(player in GetPlayerArray()){
+        if(IsPrivateMatchSpectator(player))
+        InitialisePrivateMatchSpectatorPlayer( player )
+    }
+    //end of Khalmee changes
+
+    SetGameState( eGameState.Prematch )
+}
```

Comment:
Is it just me or did the indentation on this line change? Might be an accidental tabs/spaces edit ^^

Reply:
since all of it is spaces now it needs to be changed to tabs anyway

Reply:
replaced back to tabs